This patch fixes cpp14 compilation errors (narrowing conversion).
Gentoo-bug: https://bugs.gentoo.org/show_bug.cgi?id=594896

--- a/fw/TFWCursors.cpp
+++ b/fw/TFWCursors.cpp
@@ -28,125 +28,125 @@
 
 static char kHorizCursorData[] = 
 {
-	0x00, 0x00, 
-	0x80, 0x00,
-	0xC0, 0x01,
-	0xE0, 0x03,
-	0x80, 0x00,
-	0x80, 0x00,
-	0xFE, 0x3F,
-	0x00, 0x00, 
-	0x00, 0x00, 
-	0xFE, 0x3F,
-	0x80, 0x00,
-	0x80, 0x00,
-	0xE0, 0x03,
-	0xC0, 0x01,
-	0x80, 0x00,
-	0x00, 0x00
+	(char)0x00, (char)0x00, 
+	(char)0x80, (char)0x00,
+	(char)0xC0, (char)0x01,
+	(char)0xE0, (char)0x03,
+	(char)0x80, (char)0x00,
+	(char)0x80, (char)0x00,
+	(char)0xFE, (char)0x3F,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00, 
+	(char)0xFE, (char)0x3F,
+	(char)0x80, (char)0x00,
+	(char)0x80, (char)0x00,
+	(char)0xE0, (char)0x03,
+	(char)0xC0, (char)0x01,
+	(char)0x80, (char)0x00,
+	(char)0x00, (char)0x00
 };
 
 static char kHorizCursorMask[] = 
 {
-	0x80, 0x00,
-	0xC0, 0x01,
-	0xE0, 0x03,
-	0xF0, 0x07,
-	0xE0, 0x03,
-	0xFF, 0x7F,
-	0xFF, 0x7F,
-	0xFF, 0x7F,
-	0xFF, 0x7F,
-	0xFF, 0x7F,
-	0xFF, 0x7F,
-	0xE0, 0x03,
-	0xF0, 0x07,
-	0xE0, 0x03,
-	0xC0, 0x01,
-	0x80, 0x00
+	(char)0x80, (char)0x00,
+	(char)0xC0, (char)0x01,
+	(char)0xE0, (char)0x03,
+	(char)0xF0, (char)0x07,
+	(char)0xE0, (char)0x03,
+	(char)0xFF, (char)0x7F,
+	(char)0xFF, (char)0x7F,
+	(char)0xFF, (char)0x7F,
+	(char)0xFF, (char)0x7F,
+	(char)0xFF, (char)0x7F,
+	(char)0xFF, (char)0x7F,
+	(char)0xE0, (char)0x03,
+	(char)0xF0, (char)0x07,
+	(char)0xE0, (char)0x03,
+	(char)0xC0, (char)0x01,
+	(char)0x80, (char)0x00
 };
 
 static char kVertCursorData[] = 
 {
-	0x00, 0x00, 
-	0x40, 0x02,
-	0x40, 0x02,
-	0x40, 0x02,
-	0x40, 0x02,
-
-	0x48, 0x12,
-	0x4C, 0x32,
-	0x7E, 0x7E,
-	0x4C, 0x32,
-	0x48, 0x12,
-
-
-	0x40, 0x02,
-	0x40, 0x02,
-	0x40, 0x02,
-	0x40, 0x02,
-	0x00, 0x00,
-	0x00, 0x00
+	(char)0x00, (char)0x00, 
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+
+	(char)0x48, (char)0x12,
+	(char)0x4C, (char)0x32,
+	(char)0x7E, (char)0x7E,
+	(char)0x4C, (char)0x32,
+	(char)0x48, (char)0x12,
+
+
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+	(char)0x40, (char)0x02,
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00
 };	
 	
 static char kVertCursorMask[] = 
 {
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0xE8, 0x17,
-	0xFC, 0x3F,
-	0xFE, 0x7F,
-	0xFF, 0xFF,
-	0xFE, 0x7F,
-	0xFC, 0x3F,
-	0xE8, 0x17,
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0xE0, 0x07,
-	0x00, 0x00
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0xE8, (char)0x17,
+	(char)0xFC, (char)0x3F,
+	(char)0xFE, (char)0x7F,
+	(char)0xFF, (char)0xFF,
+	(char)0xFE, (char)0x7F,
+	(char)0xFC, (char)0x3F,
+	(char)0xE8, (char)0x17,
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0xE0, (char)0x07,
+	(char)0x00, (char)0x00
 };
 
 static char kInvisibleCursorData[] = 
 {
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00
 };	
 	
 static char kInvisibleCursorMask[] = 
 {
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00,
-	0x00, 0x00, 
-	0x00, 0x00
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00,
+	(char)0x00, (char)0x00, 
+	(char)0x00, (char)0x00
 };
 
 
